examples/serve/panels-demo/config.yaml

```
@@ -0,0 +1,124 @@
+Viewer:
+  Augmentation:
+    Path: ""
+    Extensions:
+      - my_viewer_ext.ext
+
+DatasetChunkCacheSize: 100M
+
+DataStores:
+  - Identifier: opensr-test
+    StoreId: s3
+    StoreParams:
+      root: opensr-test
+    Datasets:
+      - Path: openSR_musselbeds_use_case_waddensea_S2L2A_2018-2022.levels
+        Identifier: waddensea
+        Title: Wadden Sea
+        Style: waddensea
+        Augmentation:
+          Path: compute_indexes.py
+          Function: compute_indexes
+      - Path: S2_louisville_time.levels
+        Identifier: louisville
+        Title: Lousiville S2
+        Style: louisville
+
+Styles:
+  - Identifier: waddensea
+    ColorMappings:
+      B01:
+        ColorBar: "Greys_r"
+        ValueRange: [0., 0.25]
+      B02:
+        ColorBar: "Greys_r"
+        ValueRange: [0., 0.25]
+      B03:
+        ColorBar: "Greys_r"
+        ValueRange: [0., 0.25]
+      B04:
+        ColorBar: "Greys_r"
+        ValueRange: [0., 0.25]
+      B05:
+        ColorBar: "Greys_r"
+        ValueRange: [0., 0.25]
+      B06:
+        ColorBar: "Greys_r"
+        ValueRange: [0., 0.25]
+      B07:
+        ColorBar: "Greys_r"
+        ValueRange: [0., 0.25]
+      B08:
+        ColorBar: "Greys_r"
+        ValueRange: [0., 0.25]
+      B8A:
+        ColorBar: "Greys_r"
+        ValueRange: [0., 0.25]
+      B09:
+        ColorBar: "Greys_r"
+        ValueRange: [0., 0.25]
+      B10:
+        ColorBar: "Greys_r"
+        ValueRange: [0., 0.25]
+      B11:
+        ColorBar: "Greys_r"
+        ValueRange: [0., 0.25]
+      B12:
+        ColorBar: "Greys_r"
+        ValueRange: [0., 0.25]
+      SCL:
+        ColorBar: "Accent"
+        ValueRange: [ 0, 255 ]
+      rgb:
+        Red:
+          Variable: B04
+          ValueRange: [0., 0.25]
+        Green:
+          Variable: B03
+          ValueRange: [0., 0.25]
+        Blue:
+          Variable: B02
+          ValueRange: [0., 0.25]
+
+  - Identifier: louisville
+    ColorMappings:
+      band_1:
+        ColorBar: "gray"
+        ValueRange: [0, 5000]
+      band_2:
+        ColorBar: "gray"
+        ValueRange: [0, 5000]
+      band_3:
+        ColorBar: "gray"
+        ValueRange: [0, 5000]
+      band_4:
+        ColorBar: "bone"
+        ValueRange: [0, 10000]
+      rgb:
+        Red:
+          Variable: band_1
+          ValueRange: [0., 5000]
+        Green:
+          Variable: band_2
+          ValueRange: [0., 5000]
+        Blue:
+          Variable: band_3
+          ValueRange: [0., 5000]
+
+ServiceProvider:
+  ProviderName: "Brockmann Consult GmbH"
+  ProviderSite: "https://www.brockmann-consult.de"
+  ServiceContact:
+    IndividualName: "Norman Fomferra"
+    PositionName: "Senior Software Engineer"
+    ContactInfo:
```

This info is outdated, pleas update with the following:
```suggestion
        ContactInfo:
              Phone:
                Voice: +49 40 696 389 300
                Facsimile: +49 40 696 389 399
              Address:
                DeliveryPoint: Chrysanderstr. 1
                City: Hamburg
                AdministrativeArea: Hamburg
                PostalCode: 21029
                Country: Germany
                ElectronicMailAddress: info@brockmann-consult.de
```

---

Sure :)